src/services/jobRoleMemoryService.ts

```diff
@@ -0,0 +1,13 @@
+import type { IjobRole } from "../models/job-role";
+import type { jobRoleservice } from "./interfaces";
+
+export class jobRole implements jobRoleservice {
```

Comment:
let's rename this class as well. Make sure it matches the file name too (ie, a bit about the fact that it uses job roles stored in the memory)